src/Controller/FrameworkController.php

```diff
@@ -100,4 +100,88 @@ public function index (Request $request, Session $session){
         );
         return $this->output("/framework/index.html.twig");
     }
+
+    /**
+     * @Route ("/framework-line/add", name="framework.add_line", methods={"GET"})
+     */
+    public function addLine (Request $request, Session $session){
+        $CSRFToken = $GLOBALS['CSRFSession'];
+        $id = null;
+        $nom = null;
+
+        $this->templateParams(
+            array(
+                "CSRFToken"    => $CSRFToken,
+                "id"           => $id,
+                "nom"          => $nom
```

Comment:
These 2 variables (id and nom) are useless here. When adding a new line, we never have id or nom

Reply:
They are required as parameters for the template, thus I have to declare them.